Dockerfile.ubuntu

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10.6-slim
+WORKDIR /app
+RUN apt update
+RUN apt install -y wget git python3 python3-venv libgl1 libglib2.0-0
+RUN wget -q https://raw.githubusercontent.com/AUTOMATIC1111/stable-diffusion-webui/master/webui.sh
```

Comment:
This line is unnecessary, since you're overwriting the `webui.sh` you're downloading on the very next line.

Reply:
Fixed!